c2corg_ui/static/js/apiservice.js

```diff
@@ -310,4 +310,29 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 };
 
 
+/**
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.readAccount = function() {
+  var promise = this.getJson_('/users/account');
+  promise.catch(function(response) {
+    this.alerts_.addError(response);
+  }.bind(this));
+  return promise;
```

Comment:
I considered it but decided against it since it would not gain much and is more readable this way.